source/includes/_cluster.md

```diff
@@ -0,0 +1,419 @@
+```
+
+```html
+obsolete
+```
+
+> Response Example
+```json
+{
+    "initialized": true,
+    "error": null,
+    "encryption_key": "<REDACTED>"
+}
+```
+
+`PATCH "{manager-ip}/api/v3/cluster"`
+
+Update the cluster configuration. The request body is a mapping containing
+arbitrary settings, which can be used by either the core cluster mechanisms,
+or user-specific extensions, if any.
+Only admin users can execute this operation.
+
+### Response
+
+A `ClusterState` resource.
+
+
+## The ClusterNode resource
+
+The ClusterNode resource represents the state of a node in the cluster
+
+### Attributes:
+
+Attribute | Type | Description
+--------- | ---- | -----------
+`master`| boolean | Whether this node is the current cluster master.
+`name` | string | The name of this node.
+`host_ip` | string | The externally-reachable IP of this node.
+`online` | boolean | Whether this node is currently online.
+`initialized` | boolean | Whether the node has joined the cluster successfully.
+
+
+## List Cluster Nodes
+
+> Request Example
+```python
+client.cluster.nodes.list()
+```
+
+```shell
+curl -u user:password http://<manager-ip>/api/v3/cluster/nodes
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/nodes",
+  "method": "GET",
+  "headers": headers
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+```json
+{
+    "items":
+    [
+        {
+            "initialized": true,
+            "online": true,
+            "master": true,
+            "host_ip": "172.20.0.2",
+            "name": "cloudify_manager_LMJZA2"
+        }
+    ]
+}
+```
+
+`GET "{manager-ip}/api/v3/cluster/nodes"`
+
+List all nodes in the cluster.
+
+### Response
+
+Field | Type | Description
+----- | ---- | -----------
+`items` | list | A list of `ClusterNode` resources
+
+
+## Get Cluster Node
+
+> Request Example
+```python
+client.cluster.nodes.get("<node-id>")
+```
+
+```shell
+curl -u user:password http://<manager-ip>/api/v3/cluster/nodes/<node-id>
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/nodes/<node-id>",
+  "method": "GET",
+  "headers": headers
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+```json
+{
+    "initialized": true,
+    "online": true,
+    "master": true,
+    "host_ip": "172.20.0.2",
+    "name": "cloudify_manager_LMJZA2"
+}
+```
+
+`GET "{manager-ip}/api/v3/cluster/nodes/{node-id}"`
+
+Get the details of a node in the cluster.
```

Comment:
Get the details of a node in the cluster. _Is this an instruction or a description? If it is an instruction,change `get` to `fetch`. If it is a description, change `Get` to `Fetches`._